onnxruntime/core/mlas/inc/mlas.h

```diff
@@ -1222,6 +1222,26 @@ MlasQuantizeLinear(
     OutputType ZeroPoint
     );
 
+void
+MLASCALL
+MlasQuantizeLinearU4(
+    const float* Input,
+    uint8_t* Output,
```

Comment:
nit: `Output` is to be interpreted as bytes vs 8-bit unsigned integers, right? if so, would std::byte would be clearer?